src/git/CodacyCloud.ts

```diff
@@ -9,13 +9,20 @@ import { PullRequest, PullRequestInfo } from './PullRequest'
 import { Config } from '../common/config'
 import { IssuesManager } from './IssuesManager'
 import Telemetry from '../common/telemetry'
+import { checkFirstAnalysisStatus, getRepositoryCodacyCloudStatus } from '../onboarding'
 
-export enum RepositoryManagerState {
-  NoRepository = 'NoRepository',
-  NoRemote = 'NoRemote',
+export enum CodacyCloudState {
```

Comment:
"Repository manager" sounded confusing, and I thought I could make it a stronger link with the extension view 🤔